test/controllers/projects_controller_test.rb

```diff
@@ -118,44 +118,6 @@ class ProjectsControllerTest < ActionDispatch::IntegrationTest # rubocop:disable
     assert_response :unauthorized
   end
 
-  test 'should fail to update a project with wrong params' do
```

Comment:
Can we keep this method for now, and then update/remove it when you actually change the process to rename a project or change it's path.